packages/python/plotly/setup.py

```diff
@@ -286,7 +286,7 @@ def get_latest_publish_build_info(repo, branch):
 
     url = (
         r"https://circleci.com/api/v1.1/project/github/"
-        r"{repo}/tree/{branch}?limit=10000\&filter=completed"
+        r"{repo}/tree/{branch}?limit=100&filter=completed"
```

Comment:
After I remove the `\` that makes its way into the final URL but before changing the `limit` value, this returns:
```
{
  "message" : "limit must be between 0 and 100"
}
```

I don't know what set of jobs this was returning when the `\` was there, looked recent but somehow didn't include any `publish-dist` jobs. 🤷 anyway now it seems to work locally.